src/styles/index.ts

```diff
@@ -4732,6 +4732,11 @@ const styles = (theme: ThemeColors) =>
             borderWidth: 2,
             borderColor: theme.cardBG,
             borderRadius: variables.componentBorderRadiusLarge,
+            height: 200,
```

Comment:
What was the previous height of this? I would think we want it to match what we use in the expense view just for consistency's sake.